<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormConfiguracion</class>
 <widget class="QDialog" name="FormConfiguracion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuracion</string>
  </property>
  <widget class="QComboBox" name="combo_Espacios">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>95</y>
     <width>160</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_Espacios">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>100</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Espacios Verdes</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Direccion">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>50</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Direccion de la Agencia</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Direccion">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>194</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Add">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>93</y>
     <width>30</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Aceptar">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>193</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Cancelar">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>193</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Remove">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>107</y>
     <width>30</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_Direccion</tabstop>
  <tabstop>combo_Espacios</tabstop>
  <tabstop>pb_Add</tabstop>
  <tabstop>pb_Remove</tabstop>
  <tabstop>pb_Aceptar</tabstop>
  <tabstop>pb_Cancelar</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pb_Cancelar</sender>
   <signal>clicked()</signal>
   <receiver>FormConfiguracion</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
